### ISSUE TITLE
ttnn.pad does not support negative padding (cropping)

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-metal#37475]
### Describe the bug 

- The AttentionDenseUnet model uses [negative padding operation](https://github.com/tenstorrent/tt-forge-models/blob/34ea72f658949445363b1c4dec52f0db2143c8f8/attention_denseunet/pytorch/src/model.py#L66C13-L68C14) 
- For complete context, pls checkout the discussions in https://github.com/tenstorrent/tt-xla/issues/2293
- Negative padding is supported by PyTorch but not by `ttnn.pad` - [feb6_neg_pad_cpu_run.log](https://github.com/user-attachments/files/25132903/feb6_neg_pad_cpu_run.log) 

```
Input: (1, 1024, 64, 64)
F.pad(x, [-16, -16, -16, -16]) → crops 16 pixels from each side
Output: (1, 1024, 32, 32)
```

### Test 

```
import pytest
import torch
import torch.nn.functional as F
from loguru import logger

def test_pad():

    class PadModule(torch.nn.Module):
        def __init__(self):
            super().__init__()

        def forward(self, x1):
            x1 = F.pad(x1, [-16,-16,-16,-16])
            return x1

    model = PadModule()
    model.to(torch.bfloat16)
    input = torch.randn(1, 1024, 64, 64, dtype=torch.bfloat16)
    logger.info(f"Input shape: {input.shape}")

    with torch.no_grad():
        output = model(input)

    logger.info(f"Output shape: {output.shape}")

```

**Comment by jvegaTT** [comment]
Can slice work as a potential workaround?

**Comment by kamalrajkannan78** [comment]
> Can slice work as a potential workaround?

Yes, slicing can be used as a workaround for negative padding. It is equivalent to cropping and works well. @sdjordjevicTT  could you please confirm?
